src/main/java/sirius/biz/codelists/IDBFilteredLookupTable.java

```diff
@@ -36,6 +36,10 @@
         super(extension);
         this.baseTable = baseTable;
         this.filterSet = filterSet;
+        this.codeCase = extension.get(LookupTable.CONFIG_KEY_CODE_CASE_MODE)
```

Comment:
add something similar for CustomLookup table? like, ensure that both tables use the same and use this or complain if different?